.github/workflows/DeployStaticExecutables.yaml

```diff
@@ -0,0 +1,106 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+name: Deploy static executables
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        type: string
+        required: true
+        description: The name of the release tag
+
+jobs:
+  deploy_static_execs:
+    name: Deploy static executables and libraries
+    runs-on: ubuntu-latest
+    # Running in a protected environment that provides the necessary secrets
+    environment: release
+    env:
+      RELEASE_TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
```

Comment:
Don't need this anymore when you make the change above. You can use `inputs.tag_name` directly.